src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -789,11 +793,15 @@ public function queue_import( $args = [] ) {
 				$data
 			);
 
+			tribe( 'logger' )->log_debug( "Error when the creation of the import is taking place.", 'EA Queue Import' );
```

Comment:
:no_entry_sign: String "Error when the creation of the import is taking place." does not require double quotes; use single quotes instead

<sub>hash: 3dc1579c61dc6894debea268dd94225d</sub>